newsroom/section_filters/views.py

```diff
@@ -95,15 +132,17 @@ async def edit(id):
     if validation:
         return validation
 
-    set_version_creator(updates)
-    get_resource_service("section_filters").patch(id=ObjectId(id), updates=updates)
-    return jsonify({"success": True}), 200
+    await SectionFiltersService().update(args.id, updates)
```

Comment:
We should probably catch Pydantic validation issues and return them to the client here